liota/core/package_manager.py

```diff
@@ -376,37 +359,67 @@ def run(self):
 
             # Switch on message content (command), determine what to do
             command = msg[0]
-            if command in ["load", "unload", "delete", "reload", "update"]:
+            if command in ["load", "reload", "update"]:
                 #-----------------------------------------------------------
                 # Use these commands to handle package management tasks
 
                 with package_lock:
                     if len(msg) < 2:
                         log.warning("No package is specified: %s" % command)
                         continue
-                    if len(msg) > 2:
-                        list_packages = msg[1:]
+                    if len(msg) > 3:
```

Comment:
What if length of message is equal to 2? I see a check for length < 2 and then directly for greater than 3, but no check for length = 2?